waspc/data/Generator/templates/server/src/routes/apis/index.ts

```diff
@@ -0,0 +1,37 @@
+{{={= =}=}}
+import express, { Request, Response } from 'express';
+import prisma from '../../dbClient.js'
+import { handleRejection } from '../../utils.js'
+{=# isAuthEnabled =}
+import auth from '../../core/auth.js'
+import { {= userEntityUpper =} } from '../../entities'
+{=/ isAuthEnabled =}
+
+{=# apiRoutes =}
+{=& importStatement =}
+{=/ apiRoutes =}
+
+const router = express.Router()
+
+{=# apiRoutes =}
+{=# isUsingAuth =}
+router.{= routeMethod =}('{= routePath =}', auth, handleRejection((req: Request & { user: {= userEntityUpper =} }, res: Response) => {
+{=/ isUsingAuth =}
+{=^ isUsingAuth =}
+router.{= routeMethod =}('{= routePath =}', handleRejection((req: Request, res: Response) => {
+{=/ isUsingAuth =}
+  const context = {
+    {=# isUsingAuth =}
+    user: req.user,
```

Comment:
In operations, we strip the `password` filed. Do we want to do the same here?

Reply:
Good question. It gets stripped out in the middleware that ops and apis use: https://github.com/wasp-lang/wasp/blob/main/waspc/data/Generator/templates/server/src/core/auth.js#L47-L53

Just `console.log`ged it to confirm and no password in api context 👍🏻

Reply:
In that case, we should correct the type. We're currently telling users they're working with `User`. You can omit the password field like this: https://github.com/wasp-lang/wasp/blob/cb691c8dd7348553bee123b2cdcf05d75847a0b5/waspc/data/Generator/templates/server/src/_types/index.ts#L28 (mind the comment :)).